### DISCUSSION TITLE
Błąd: Full authentication is required to access this resource (sandbox)

### DISCUSSION THREAD

**Discussion by ryszardlet** [allegro/allegro-api#10420] [Suggestions]
Podczas prób odczytywania zasobów _sale/responsible-persons_ dostaję błąd:
`{"error":"unauthorized","error_description":"Full authentication is required to access this resource"}`
trace-id: b3ee319431d65f03

Myślałem, że to kwestia uprawnień aplikacji, ale stosownych ustawień nie znalazłem. Co takiego jest przyczyną, że taki błąd się pojawił?

**Comment by MaciejFrackowiak** [comment]
Odpowiedziałem w wątku #10425, ponieważ nie jest to sugestia.
